### ISSUE TITLE
Firebase Full Implementation Method

### ISSUE THREAD

**Issue by carl-neng** [EricPucho/FirebaseXamarinForms#1]
Hello Sir. Why you do not continue implement all method or event for firebase on this project. I want to see this project full option for firebase such as: Child Change, Push notification,....... I hope full this good project for xamarin forms with firebase. I'm really thank to create sample this for xamarin forms.

**Comment by EricPucho** [comment]
Hi, 
This sample in based on FireSharp plugin. Unfortunately, https://github.com/ziyasal/FireSharp
doesn't already implement this features.
But stay ready ;-)

**Comment by carl-neng** [comment]
Yes sound good to hear that. I'm waiting..... I hope so that see it soon......Thank advance
